sdk/include/reactos/shlwapi_undoc.h

```diff
@@ -203,6 +202,12 @@ SHCreatePropertyBagOnProfileSection(
     _In_ DWORD dwMode,
     _In_ REFIID riid,
     _Out_ void **ppvObj);
+EXTERN_C HRESULT WINAPI
```

Comment:
```suggestion

EXTERN_C HRESULT WINAPI
```
(same reason as above.)